surround_cli/requirements.txt

```diff
@@ -1,7 +1,6 @@
-pyaml==17.12.1
-doit==0.31.1
-pandas==0.25.2
 numpy==1.17.3
+pandas==0.25.2
+doit==0.31.1
 tornado==6.0.2
 google-cloud-storage==1.20.0
-pylint==2.4.3
\ No newline at end of file
+surround==0.0.14
```

Comment:
You still  need  `pylint` as it is a dependency of the linter.

Reply:
Good find, added back